### ISSUE TITLE
Crash on quit

### ISSUE THREAD

**Issue by Fulons** [nudge-reality/bluetooth-windows-rt#7]
### Expected behaviour

When calling quit all BLE communication should stop and device should be free to "connect" again.

### Actual behaviour

When trying to quit/shutdown the BLE library from Unity it crashes. Unity editor just shuts down with the BLE thread still running in the background. 
Suspecting something is not right when cleaning up the recently added read data.

### Steps to Reproduce

Call `BleApi.Quit();`

### Screenshots or Logs

_N/A_

### Additional Context

_N/A_

**Comment by SamHignett** [comment]
@Fulons Is this issue still relevant?

**Comment by Fulons** [comment]
Yes it is relevant, but not urgent. It just means that every time you shut down the backend Unity just crashes and shuts down. Everything implemented so far works in editor without running, which is why I would not deem it urgent.

It is however important to fix at some point.
